pkg/util/runtime/features.go

```diff
@@ -100,6 +100,15 @@ var (
 	// * change the value in site/content/en/docs/Guides/feature-stages.md.
 	// * Ensure that the features in each file are organized categorically and alphabetically.
 	//
+	// Feature Promotion: Alpha to Beta for SDK Functions
+	// * Move alpha feature from alpha files to beta files across the SDKs.
```

Comment:
Is this step refering the alpha.proto and beta.proto files?

Reply:
I've generalized this sentence. We also need to move from `Alpha.cs` to `Beta.cs`, from `IAgonesAlphaSDK.cs` to `IAgonesBetaSDK.cs`, and from `alpha.go` to `beta.go`. Would it be easier if we referred to the files more generally, like alpha.proto to beta.proto and alpha.go to beta.go?

Reply:
It was a little unclear to me which bullet points were referring to the proto, the SDK server, and the SDK client files. Maybe make that a little bit clearer, so the next person doing this know what general areas to modify?

Reply:
I have modified the steps, please share your thoughts..